### DISCUSSION TITLE
vCenter vsphere support?

### DISCUSSION THREAD

**Discussion by johnfelipe** [apache/hertzbeat#280] [Q&A]
Is possible support VMware environment?

**Comment by tomsun28** [comment]
@johnfelipe Of course, we consider using vijava sdk or rest api to monitor esx and vcenter later.

**Reply by johnfelipe** [reply]
But Right now?
Is any method for do it, or if u know other open source repo for do it meanwhile.

**Reply by tomsun28** [reply]
I don't know about the other repo.

**Comment by johnfelipe** [comment]
When you release vsphere vCenter compatibility?

**Reply by tomsun28** [reply]
If it goes well, we try to do it in 1 month or something. Added to roadmap.
